ckanapi/cli/main.py

```diff
@@ -4,7 +4,13 @@
   ckanapi action ACTION_NAME
           [KEY=VALUE ... | -i | -I JSON_INPUT] [-j | -J]
           [[-c CONFIG] [-u USER] | -r SITE_URL [-a APIKEY] [-g]]
-  ckanapi load (datasets | groups | organizations | users | related)
+  ckanapi load (datasets)
```

Comment:
no need for () around "datasets" here, and why not move the [--upload-resources] up to the top line so it's more obvious what option is special for loading datasets. Same for load (groups | organizations)